libs/constant/events.constant.ts

```diff
@@ -0,0 +1,41 @@
+type UpcomingEvents = {
+    image: string;
+    alt: string;
+    title: string;
+    des: string;
+  };
+  export const upcomingEvents : UpcomingEvents[] = [
+    {
+      image: "/detail.jpeg",
+      alt: "detail",
+      title:"DtaleD",
+      des:"Join us for an inspiring session with Thajudeen Aboobaker, CEO of UrbanTrash, on August 9, 2024. Gain insights into his transformative journey and engage in interactive discussions."
+    },
+    {
+      image: "/maker.jpeg",
+      alt: "maker",
+      title:"Maker Fair",
+      des:"Join the 𝗠𝗮𝗸𝗲𝗿 𝗙𝗮𝗶𝗿 on August 9, 2024, at Christ College of Engineering to showcase, sell, and network with creative minds. Inaugurated by Mr. Thajudeen Aboobaker."
+    },
+  ];
+
+  type Events = {
```

Comment:
Type Events and  UpcomingEvents are redundant, keep a single type named event inside types -> global.types